src/os/ui/slick/slickgrid.js

```diff
@@ -452,6 +437,53 @@ os.ui.slick.SlickGridCtrl.prototype.disposeInternal = function() {
 };
 
 
+/**
+ * @param  {boolean} newValue
+ * @param  {boolean=} oldValue
```

Comment:
The `oldValue` parameter here should have be `opt_oldValue` since it's an optional parameter. This should be enforced by the compiler or the linter as well...